nautobot/docs/administration/nautobot-server.md

```diff
@@ -0,0 +1,385 @@
+$ nautobot-server runjob local/example/MyJobWithVars --commit
+```
+
+Please see the [guide on Jobs](../additional-features/jobs.md) for more information on orking with and running jobs.
+
+### `start`
+
+`nautobot-server start`
+
+Directly invoke uWSGI to start a Nautobot server suitable for production use. This command behaves exactly as uWSGI does, but allows us to maintain a single entrypoint into the Nautobot application.
+
+!!! note
+    uWSGI offers an overwhelming amount of command-line arguments that could not possibly be covered here. Please see the [official uWSGI Options guide](https://uwsgi-docs.readthedocs.io/en/latest/Options.html) for more information.
+
+```no-highlight
+$ nautobot-server start --ini ./uwsgi.ini
+[uWSGI] getting INI configuration from ./uwsgi.ini
+[uwsgi-static] added mapping for /static => /opt/nautobot/static
+*** Starting uWSGI 2.0.19.1 (64bit) on [Thu Mar 11 21:13:22 2021] ***
+compiled with version: 8.3.1 20190311 (Red Hat 8.3.1-3) on 23 September 2020 02:39:40
+os: Linux-5.4.0-52-generic #57-Ubuntu SMP Thu Oct 15 10:57:00 UTC 2020
+nodename: jathan-nautobot-testing
+machine: x86_64
+clock source: unix
+pcre jit disabled
+detected number of CPU cores: 48
+current working directory: /opt/nautobot
+detected binary path: /usr/bin/python3.8
+your processes number limit is 256070
+your memory page size is 4096 bytes
+detected max file descriptor number: 1024
+building mime-types dictionary from file /etc/mime.types...567 entry found
+lock engine: pthread robust mutexes
+thunder lock: disabled (you can enable it with --thunder-lock)
+uwsgi socket 0 bound to TCP address 0.0.0.0:9191 fd 7
+Python version: 3.8.5 (default, Jan 27 2021, 15:41:15)  [GCC 9.3.0]
+--- Python VM already initialized ---
+Python main interpreter initialized at 0x2573e30
+python threads support enabled
+your server socket listen backlog is limited to 1024 connections
+your mercy for graceful operations on workers is 60 seconds
+mapped 636432 bytes (621 KB) for 15 cores
+*** Operational MODE: preforking+threaded ***
+WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x2573e30 pid: 112153 (default app)
+spawned uWSGI master process (pid: 112153)
+spawned uWSGI worker 1 (pid: 112159, cores: 3)
+spawned uWSGI worker 2 (pid: 112162, cores: 3)
+spawned uWSGI worker 3 (pid: 112165, cores: 3)
+spawned uWSGI worker 4 (pid: 112168, cores: 3)
+spawned uWSGI worker 5 (pid: 112171, cores: 3)
+```
+
+Please see the guide on [Deploying Nautobot](../installation/wsgi.md) for our recommended configuration for use with uWSGI.
+
+### `trace_paths`
+
+`nautobot-server trace_paths`
+
+Generate any missing cable paths among all cable termination objects in Nautobot.
+
+After upgrading the database or working with Cables, Circuits, or other related objects, there may be a need to rebuild cached cable paths.
+
+`--force`<br>
+Force recalculation of all existing cable paths.
+
+`--no-input`<br>
+Do not prompt user for any input/confirmation.
```

Comment:
What/when does it normally prompt the user for?

Reply:
It looks like that only matters if you use `--force`; `--no-input` suppresses the confirmation prompt.

These help strings were lifted directly from the source code and CLI outputs.

Reply:
I figured that was the case, was just wondering if it'd be useful to improve these strings. No worries.